internal/entities/quote/common.go

```diff
@@ -44,6 +46,38 @@ func ValidateQuoteHash(hash string) error {
 }
 
 func CalculateCallFee(amount *entities.Wei, config PegConfiguration) *entities.Wei {
-	// TODO implement in GBI-2528
-	return entities.NewWei(100000000000)
+	result := new(entities.Wei)
+
+	percentageFee := calculatePercentageFee(amount, config.GetFeePercentage())
```

Comment:
If the config GetFeePercentage() return nil or invalid the method can panic.

Reply:
thanks, just added those cases to the test suite